src/mapcraftercore/renderer/blocktextures.cpp

```diff
@@ -11,33 +11,112 @@ namespace mapcrafter {
 namespace renderer {
 
 BlockTextures::BlockTextures()
-	: ANVIL_BASE("anvil_base"),
-	  ANVIL_TOP_DAMAGED_0("anvil_top_damaged_0"),
-	  ANVIL_TOP_DAMAGED_1("anvil_top_damaged_1"),
-	  ANVIL_TOP_DAMAGED_2("anvil_top_damaged_2"),
+	: ACACIA_DOOR_BOTTOM("acacia_door_bottom"),
```

Comment:
Just FYI, this doesn't need to be updated as Mapcrafter doesn't need the Minecraft texture files anymore (but only the generated block images instead). We can keep it as it is now though.